Add option to disable verifying HTTPS certificates (#38)
xperimental/nextcloud-exporter: README.md

go get github.com/xperimental/nextcloud-exporter
```

> or with Docker:

Reviewer: I agree that the readme is a bit outdated by now and needs an update, but I would leave it out of this PR except for the updates describing the current change.

Author: Alright.

```plain
$ nextcloud-exporter --help
Usage of nextcloud-exporter:
  -a, --addr string            Address to listen on for connections. (default ":9205")
  -c, --config-file string     Path to YAML configuration file.
      --login                  Use interactive login to create app password.
  -p, --password string        Password for connecting to Nextcloud.
  -s, --server string          URL to Nextcloud server.
  -t, --timeout duration       Timeout for getting server info document. (default 5s)

>   -u, --username string        Username for connecting to Nextcloud.
>   --tls-skip-verify boolean    Disables checking HTTPS certificates of Nextcloud.

Reviewer: I usually use the actual output of the tool to create this. Noticed that I forgot to update this with the last PR:

```suggestion
      --tls-skip-verify      Skip certificate verification of Nextcloud server.
  -u, --username string      Username for connecting to Nextcloud.
  -V, --version              Show version information and exit.
```

|       `NEXTCLOUD_PASSWORD` | --password      |
| `NEXTCLOUD_LISTEN_ADDRESS` | --addr          |
|        `NEXTCLOUD_TIMEOUT` | --timeout       |

> |  `NEXTCLOUD_TLS_NO_VERIFY` | --tls-no-verify |

Reviewer: This still has the old name. Unfortunately this also means that the table needs to be reformatted...

```suggestion
| `NEXTCLOUD_TLS_SKIP_VERIFY` | --tls-skip-verify |
```

# optional
listenAddress: ":9205"
timeout: "5s"

> tlsNoVerify: false

Reviewer: Still has the old name:

```suggestion
tlsSkipVerify: false
```
